databricks/sdk/mixins/files.py

```diff
@@ -267,7 +267,8 @@ def __repr__(self) -> str:
 class _Path(ABC):
 
     def __init__(self, path: str):
-        self._path = pathlib.Path(str(path).replace('dbfs:', '').replace('file:', ''))
+        self._path = pathlib.PurePosixPath(str(path).replace('dbfs:', '').replace('file:', ''))
+        self._system_path = pathlib.Path(self._path)
```

Comment:
It'd be good to add a comment explaining why we need `self._path` to be `PurePosixPath` and why we need `self._system_path`